adbms/src/main/scala/de/up/hpi/informationsystems/adbms/Dactor.scala

```diff
@@ -16,6 +16,16 @@ object Dactor {
   def dactorOf(factory: ActorRefFactory, clazz: Class[_ <: Dactor], id: Int): ActorRef =
     factory.actorOf(Props(clazz, id), nameOf(clazz, id))
 
+  /**
+    * Looks up the path to a Dactor and returns the `ActorSelection`.
+    * @note lookup is global to the system, i.e. /user/$dactorName
+    * @param clazz class of the Dactor
+    * @param id id of the Dactor
+    * @return ActorSelection of the lookup
+    */
+  def dactorSelection(system: ActorSystem, clazz: Class[_ <: Dactor], id: Int): ActorSelection =
+    system.actorSelection(s"/user/${nameOf(clazz, id)}")
```

Comment:
if you do not use the `ActorSystem` for creating the path, i think you can go back to `ActorRefFactory` trait

Reply:
Good call! I'll push that change immediately.